README.md

```diff
@@ -68,6 +68,7 @@ Polyfills are provided for:
 - the `Date*Exception/Error` classes introduced in PHP 8.3;
 - the `SQLite3Exception` class introduced in PHP 8.3;
 - the `mb_ucfirst` and `mb_lcfirst` functions introduced in PHP 8.4;
+- the `grapheme_str_split` function introduced in PHP 8.4 (requires PHP >= 7.3);
```

Comment:
I'd remove the need for PHP 7.3 as that makes the polyfill dangerous IMHO
Instead, we can borrow from SYMFONY_GRAPHEME_CLUSTER_RX as defined in the intl-grapheme polyfill
we can copy/paste the string I think